docs/src/man/man3/hal_add_funct_to_thread.3.adoc

```diff
@@ -31,10 +31,10 @@ position::
 
 == DESCRIPTION
 
-*hal_add_funct_to_thread* adds a function exported by a realtime HAL component to a realtime thread.
+The function *hal_add_funct_to_thread* adds another function that is exported by a realtime HAL component to a realtime thread.
```

Comment:
```suggestion
The function *hal_add_funct_to_thread* adds a function that is exported by a realtime HAL component to a realtime thread.
```

Reply:
I consider the "another" important for the understanding.